### PR TITLE
Log the error from the Metadata API

### PR DESCRIPTION
**Pull request by cbaines**
Also, refactor the show method to make it simplier by moving the use
of the metadata api response inside the begin/rescue/end block.

### CODE REVIEW

File: app/controllers/info_controller.rb [hunk 1]
```diff
@@ -11,23 +11,22 @@ class InfoController < ApplicationController
...
+
       document_type = @content["document_type"] || @content["format"]
 
       part_urls = get_part_urls(@content, @slug)
       @is_multipart = is_multipart(part_urls, document_type)
       calculated_metrics = metrics_from(metadata.fetch("performance"), part_urls, @is_multipart)
       @lead_metrics = calculated_metrics[:lead_metrics]
       @per_page_metrics = calculated_metrics[:per_page_metrics]
+    rescue StandardError => e
+      logger.error "Error fetching metadata for #{@slug} from the Metadata API"
```
Comment (bevanloon) [inline app/controllers/info_controller.rb]: my only pedantic comment is that the error message is perhaps a bit too specific - the fetching of the metadata is really only happening in line 18. As far as I can tell, the intent of this action is to ultimately consume the metadata and create metrics. Perhaps an error message to that effect would be more descriptive of what could possibly have gone wrong?

Comment (cbaines) [inline app/controllers/info_controller.rb]: e.g. something like "Metadata related error for #{slug}"?

Comment (cbaines) [inline app/controllers/info_controller.rb]: I've changed the message now.

**Comment by cbaines** [comment]
Thanks for you review Bevan :)

**Pull request merged**